packages/webapp/src/components/Modals/ModalComponent/v1/index.jsx

```diff
@@ -37,6 +40,7 @@ export default function ModalComponent({
       >
         {buttonLabel}
       </Button>
+      {error && <span className={styles.error}>{error}</span>}
```

Comment:
I'd suggest using the `Error` component from the `Typography` file here which will already give you the right styling (the font color is a deeper red than the regular red used here), and then using custom styling only for the margin.

Reply:
Thanks for the suggestion - I went ahead and replaced it with the Error component.

Reply:
Thank you for updating this and for the contribution! 🙂